### PR TITLE
Post release

### PR DESCRIPTION
**Pull request by mnonnenmacher**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,13 @@
...
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## 3.0.0 (under development)
+## 3.1.0 (under development)
+
+## 3.0.0 (2018-01-10)
 
 - Add support for auto-generated DSL and Job DSL extensions by executing the Job DSL scripts in a local Jenkins
   instance.
+- PipelineBuilder: Do not overwrite values with default configuration.
```
Comment (sschuberth) [inline CHANGELOG.md]: How about referring to issue numbers (if any) here?

Comment (mnonnenmacher) [reply CHANGELOG.md]: Done.

Comment (sschuberth) [reply CHANGELOG.md]: Uhm, and have links? 😉

Comment (mnonnenmacher) [reply CHANGELOG.md]: Aren't ticket numbers automatically converted to links?
https://help.github.com/articles/autolinked-references-and-urls/#issues-and-pull-requests

Comment (sschuberth) [reply CHANGELOG.md]: > References to URLs, issues, pull requests, and commits are automatically shortened and converted into links.

So, not in `README.md`.

Comment (mnonnenmacher) [reply CHANGELOG.md]: You are right. I will improve this in another commit later and merge this now so that you can rebase your other commit.

**Pull request merged**